Code/Editor/Settings.cpp

```diff
@@ -848,6 +849,7 @@ void SEditorSettings::PostInitApply()
     REGISTER_CVAR2_CB("ed_toolbarIconSize", &gui.nToolbarIconSize, gui.nToolbarIconSize, VF_NULL, "Override size of the toolbar icons 0-default, 16,32,...", ToolbarIconSizeChanged);
 
     REGISTER_CVAR2("ed_backgroundUpdatePeriod", &backgroundUpdatePeriod, backgroundUpdatePeriod, 0, "Delay between frame updates (ms) when window is out of focus but not minimized. 0 = disable background update. -1 = update with no delay.");
+    REGISTER_CVAR2("ed_backgroundSystemTickFPS", &backgroundSystemTickFPS, backgroundSystemTickFPS, 30, "FPS at which SystemTick occurs when Editor is backgrounded and frame updates are disabled.");
```

Comment:
may I suggest we keep the time unit consistent? either use MS or FPS but not mix them?  I'd prefer MS like ed_backgroundUpdatePeriod uses FWIW.

Reply:
Sure, I find FPS easier to reason about but it's relatively minor.